src/Goteo/Model/Filter.php

```diff
@@ -741,7 +741,7 @@ public function getDonors($offset = 0, $limit = 0, $count = false, $lang = null)
                     $sqlFilter .= " AND YEAR(CURDATE())";
                 }
             } else if (isset($this->enddate)) {
-                $sqlFilter .= " AND donor.year <= :endyear ";
+                $sqlFilter .= " AND donor_cert.year <= :endyear ";
```

Comment:
I've taken the chance to fix some issues with this method since it threw SQL errors. I would gladly show them to you.